packages/web-client/app/components/card-pay/layer-one-connect-modal/index.css

```diff
@@ -1,4 +1,26 @@
+.layer-one-connect-modal__scroll-wrapper {
+  height: 100%;
+  margin: 0 var(--boxel-sp-sm);
+  padding: var(--boxel-sp-lg) var(--boxel-sp-sm);
+  overflow-y: auto;
+}
+
+.layer-one-connect-modal__card:focus {
+  outline-color: transparent;
+  outline-offset: -5px;
+  box-shadow: 0 0 0 2px var(--boxel-blue);
+}
+
+.layer-one-connect-modal__card:focus:not(:focus-visible) {
+  box-shadow: none;
+}
+
+
+/* remove */
 .layer-one-connect-modal__card {
-  max-height: 48rem;
+  min-height: 40rem;
+  padding: var(--boxel-sp-lg);
 }
 
+
+/* TODO: Close button */
```

Comment:
Should this comment be left in? The design didn't have a close button AFAIK.

Reply:
There is an X in the top right in the design that's labelled "Modal Version". Might be a later version of the design.